tests/dummy/app/templates/overview.hbs

```diff
@@ -103,7 +103,7 @@ $ node example.js</code></pre>
     <div class="col-md-6">
       <hr>
       <h1>Changelog</h1>
-      <p>The current version is 0.4.0.</p>
+      <p>The current version is 0.5.1.</p>
```

Comment:
This one was not updated manually. It uses the version in package.json, and it was built when I ran grunt.